tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -226,7 +243,7 @@ public function test_process_redirect_payment_returns_valid_response() {
 		$this->assertEquals( 'processing', $final_order->get_status() );
 		$this->assertEquals( 'Credit card / debit card', $final_order->get_payment_method_title() );
 		$this->assertEquals( $payment_intent_id, $final_order->get_meta( '_stripe_intent_id', true ) );
-		$this->assertEquals( true, $final_order->get_meta( '_stripe_upe_redirect_processed', true ) );
+		$this->assertTrue( (bool) $final_order->get_meta( '_stripe_upe_redirect_processed', true ) );
```

Comment:
I thought it looked nicer to use `assertTrue` here, but unfortunately even though the value that gets saved to the meta field is `true`, it gets stored as a truthy `1` instead (thus, the boolean cast is required). `assertEquals` does not perform strict type checking apparently, which is why the previous test also works.